semgrep-core/src/core/Rule.ml

```diff
@@ -209,6 +209,7 @@ type extract_spec = {
   dst_lang : Xlang.t;
   (* e.g., $...BODY, $CMD *)
   extract : MV.mvar;
+  transform : extract_transform;
```

Comment:
We need to make this an 'extract_transform option' or add a NoTransfo in extract_transform.

Reply:
should I also remove the "unquote_string" then? I thought that was the "do nothing" option.

Reply:
no it's meant for scenario that I remember where we extract code, but the extracted code is a big string and
to make it valid we need to unquote it.

Reply:
Done.